celer/PN_logreg.pyx

```diff
@@ -110,7 +110,8 @@ def newton_celer(
             is_sparse, theta, X, X_data, X_indices, X_indptr,
             screened, X_mean, weights_pen, center, positive)
 
-        if norm_Xtheta > 1.:
+        # alpha instead of 1
+        if norm_Xtheta > alpha:
```

Comment:
if the norm is greater than alpha, you should divide by norm(X.T @ theta) / alpha, not by norm(X.T @ theta)